akka-persistence-testkit/src/main/scala/akka/persistence/testkit/internal/Unpersistent.scala

```diff
@@ -0,0 +1,465 @@
+/*
+ * Copyright (C) 2022 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.testkit.internal
+
+import akka.actor.typed.Behavior
+import akka.actor.typed.internal.BehaviorImpl.DeferredBehavior
+import akka.actor.typed.scaladsl.{ AbstractBehavior, ActorContext, Behaviors }
+import akka.annotation.InternalApi
+import akka.persistence.testkit.{ javadsl, scaladsl }
+import akka.persistence.typed.internal.EventSourcedBehaviorImpl
+import akka.persistence.typed.internal.Running.WithSeqNrAccessible
+import akka.persistence.typed.state.internal.DurableStateBehaviorImpl
+import akka.persistence.typed.state.internal.Running.WithRevisionAccessible
+import akka.util.ConstantFun.{ scalaAnyToUnit => doNothing }
+import akka.util.ccompat.JavaConverters.{ SeqHasAsJava, SetHasAsJava, SetHasAsScala }
+
+import scala.annotation.tailrec
+import scala.collection.immutable
+import scala.collection.mutable.ListBuffer
+import scala.reflect.ClassTag
+
+import java.util.concurrent.ConcurrentLinkedQueue
+
+/*
```

Comment:
it should be `/**` so that it shows in api docs (if those internals show up that). Anyway, we have `/**` in all other places afaik